docs/platforms/javascript/common/install/esm-without-import.mdx

```diff
@@ -20,10 +20,12 @@ supported:
 When running your application in ESM mode, you will most likely want to <PlatformLink to="/install/esm">follow the ESM instructions</PlatformLink>. However, if you want to avoid using the `--import` command line option, for example if you have no way of configuring a CLI flag, you can also follow an alternative setup that involves importing the `instrument.mjs` file directly in your application.
 
 <Alert level='warning' title='Restrictions of this installation method'>
-This installation method has a fundamental restriction: It only supports limited performance instrumentation. Only basic `http` instrumentation will work, and no DB or framework-specific instrumentation will be available.
-
-Because of this, we recommend using this only if the `--import` flag is not an option for you.
+  This installation method has fundamental restrictions:
+  - It only supports limited performance instrumentation.
+  - Only basic `http` instrumentation will work.
+  - No DB or framework-specific instrumentation will be available.
 
+  We recommend using this only if the `--import` flag is not an option for you.
```

Comment:
l: should we link to the --import guide? I know we reference it above but users might not have read it and jumped directly to the alert